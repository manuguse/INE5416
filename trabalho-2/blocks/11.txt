1 1 6 6 6 6 6 16 16 16
1 1 7 6 6 12 12 12 19 19
2 2 7 11 11 12 15 15 15 21
2 3 3 8 11 11 11 15 20 21
3 3 8 8 8 11 15 15 21 21
3 3 8 9 9 9 15 17 17 17
4 4 9 9 9 13 13 13 17 22
5 4 10 10 10 13 13 13 22 22
5 4 4 10 10 14 13 18 22 22
5 5 5 5 10 14 14 18 22 22