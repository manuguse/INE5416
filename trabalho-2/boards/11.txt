4 - 7 - 4 6 3 - 2 3
- - - 1 - 4 - 2 - 2
2 - - 4 - - 7 - - 4
- 6 - - - 6 - 4 - 3
5 - 5 - 1 - 6 - - 1
4 1 - 3 - 4 2 - - -
- - 1 - - 7 - 3 - 7
5 3 - 5 6 - 5 - 6 3
3 - 4 - - - - - - -
1 - 6 4 3 - 2 - 4 -