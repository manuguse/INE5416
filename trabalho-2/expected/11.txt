4 2 7 5 4 6 3 1 2 3
3 1 2 1 2 4 1 2 1 2
2 3 1 4 5 3 7 5 3 4
1 6 2 3 2 6 1 4 1 3
5 3 5 2 1 3 6 1 2 1
4 1 4 3 6 4 2 4 3 2
2 5 1 2 5 7 6 3 1 7
5 3 2 5 6 2 5 1 6 3
3 1 4 1 4 3 4 2 5 2
1 2 6 4 3 1 2 1 4 1